quiche/src/tests.rs

```diff
@@ -94,6 +96,38 @@ fn transport_params() {
     assert_eq!(new_tp, tp);
 }
 
+#[test]
+fn transport_params_decode_version_information() {
+    // Given an encoded version_information transport parameter, ...
+    let raw_version_information_params = [
+        0x11, 0x10, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x01, 0xfa, 0x5a,
+        0x0a, 0x7a, 0xff, 0x00, 0x00, 0x1d,
+    ];
```

Comment:
Would be good to have some tests for the error cases where this input is truncated in the middle of the chosen version or one of the other versions.

Reply:
I agree and will do!!